tests/Fixer/LanguageConstruct/SingleSpaceAfterConstructFixerTest.php

```diff
@@ -0,0 +1,421 @@
+
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFixWithReturnCases()
+    {
+        return [
+            [
+                '<?php return 9000; ?>',
+            ],
+            [
+                '<?php return 9000; ?>',
+                '<?php return  9000; ?>',
+            ],
+            [
+                '<?php return /* foo */9000; ?>',
+                '<?php return/* foo */9000; ?>',
+            ],
+        ];
+    }
+
+    /**
+     * @dataProvider provideFixWithThrowCases
+     *
+     * @param string      $expected
+     * @param null|string $input
+     */
+    public function testFixWithThrow($expected, $input = null)
+    {
+        $this->fixer->configure([
+            'constructs' => [
+                'throw',
+            ],
+        ]);
+
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFixWithThrowCases()
+    {
+        return [
+            [
+                '<?php throw new Exception(); ?>',
+            ],
+            [
+                '<?php throw new Exception(); ?>',
+                '<?php throw  new Exception(); ?>',
+            ],
+            [
+                '<?php throw /* foo */new Exception(); ?>',
+                '<?php throw/* foo */new Exception(); ?>',
+            ],
+        ];
+    }
+
+    /**
+     * @dataProvider provideFixWithYieldCases
+     *
+     * @param string      $expected
+     * @param null|string $input
+     */
+    public function testFixWithYield($expected, $input = null)
+    {
+        $this->fixer->configure([
+            'constructs' => [
+                'yield',
+            ],
+        ]);
+
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFixWithYieldCases()
+    {
+        return [
+            [
+                '<?php function foo() { yield "Foo"; } ?>',
+            ],
+            [
+                '<?php function foo() { yield "Foo"; } ?>',
+                '<?php function foo() { yield  "Foo"; } ?>',
+            ],
+            [
+                '<?php function foo() { yield /* foo */"Foo"; } ?>',
+                '<?php function foo() { yield/* foo */"Foo"; } ?>',
+            ],
+        ];
+    }
+
+    /**
+     * @requires PHP 7.0
+     * @dataProvider provideFixWithYieldFromCases
+     *
+     * @param string      $expected
+     * @param null|string $input
+     */
+    public function testFixWithYieldFrom($expected, $input = null)
+    {
+        $this->fixer->configure([
+            'constructs' => [
+                'yield_from',
+            ],
+        ]);
+
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFixWithYieldFromCases()
+    {
+        return [
+            [
+                '<?php function foo() { yield from [2, 12, 13]; } ?>',
+            ],
+            [
+                '<?php function foo() { yield from [2, 12, 13]; } ?>',
+                '<?php function foo() { yield from  [2, 12, 13]; } ?>',
```

Comment:
do we have an multiline example like
```php
echo
        123;
```
?

Reply:
Added!